src/vivarium_gates_mncnh/components/neonatal_causes.py

```diff
@@ -85,4 +88,15 @@ def get_normalized_csmr(self, index: pd.Index) -> pd.Series:
 
         normalized_csmr = super().get_normalized_csmr(index)
         normalized_csmr.loc[ga_greater_than_37] = 0
+        # Weight csmr for preterm birth with rds
+        if self.neonatal_cause == NEONATAL_CAUSES.PRETERM_BIRTH_WITH_RDS:
```

Comment:
If csmr is a rate here is it still ok to multiply it by a probability since we are "weighting" the csmr to get the correct number of cases?

Reply:
Yes, because the weighted rates will sum to the total rate.
```
rate * weight + rate * (1 - weight) 
    = rate * (weight + 1 - weight)
    = rate
```